brozzler/__init__.py

```diff
@@ -146,7 +121,9 @@ def behavior_script(url, template_parameters=None, behaviors_dir=None):
     """
     Returns the javascript behavior string populated with template_parameters.
     """
-    import re, logging, json
+    import re, json
+
+    logger = structlog.get_logger()
```

Comment:
I think we probably want to pass `__name__` to `get_logger` across the board.

Reply:
Yeah, good call. I'll update the `get_logger()` calls that are missing `logger_name` to make sure they get something appropriate.